JDI.Light/JDI.Light/Factories/WebDriverFactory.cs

```diff
@@ -163,11 +163,20 @@ public void SetRunType(string runType)
                 case "remote":
                     RunType = RunType.Remote;
                     return;
+                default:
+                {
+                    DoNothing();
+                    break;
+                }
             }
 
             RunType = RunType.Local;
         }
 
+
+        private void DoNothing()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add a nested comment explaining why this method is empty, throw a 'NotSupportedException' or complete the implementation.](https://app.codacy.com/manual/42_/jdi-light-csharp/pullRequest?prid=5686813)